.github/PULL_REQUEST_TEMPLATE/pull_request_template.md

```diff
@@ -0,0 +1,47 @@
+<!-- Thank you for your pull request! Please provide a description of the changes in this PR in the field above and provide the following information. -->
+
+Please check if your PR fulfills the following requirements. This is simply a reminder of what we are going to look for before merging your PR.
```

Comment:
Add `If you don't know all of this information when you create your PR don't worry - you can edit this template as you're working on it`

Reply:
Done.